### PR TITLE
Ft ensure use of strong params

### PR DESCRIPTION
**Pull request by andela-ooranagwa**
Fixes [#203]

### CODE REVIEW

File: app/controllers/local_resources_controller.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@ def create
     end
 
     update_local_resource_categories
+    binding.pry
```
Comment (davidleach) [inline app/controllers/local_resources_controller.rb]: Could you remove this call please?

File: app/controllers/work_schedule_controller.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def delete
   end
 
   def show
-    @work_schedule = WorkSchedule.find(params[:id])
+    @work_schedule = WorkSchedule.find_by(id: params[:id])
     @users = User.all
   end
```
Comment (davidleach) [inline app/controllers/work_schedule_controller.rb]: We liked how you used a before_action on the EventController.  Could you do that here to set work_schedule?

**Comment by davidleach** [comment]
Thanks again for the PR.

File: app/controllers/local_resources_controller.rb (hunk 1 above)
Comment (andela-ooranagwa) [inline app/controllers/local_resources_controller.rb]: Damn! Thanks for spotting it. I'll fix it asap.

File: app/controllers/work_schedule_controller.rb (hunk 2 above)
Comment (andela-ooranagwa) [inline app/controllers/work_schedule_controller.rb]: Ok. Will do.

**Comment by jjulien** [comment]
Looks good, thanks!

**Pull request merged**